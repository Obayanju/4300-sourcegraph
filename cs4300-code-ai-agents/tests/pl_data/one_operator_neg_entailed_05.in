tell X & Y
dpll !X
