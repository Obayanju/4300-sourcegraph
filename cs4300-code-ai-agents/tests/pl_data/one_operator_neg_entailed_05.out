NO, !X is not entailed.
